openwisp_utils/admin_theme/email.py

```diff
@@ -11,16 +11,27 @@
 logger = logging.getLogger(__name__)
 
 
-def send_email(subject, body_text, body_html, recipients, extra_context={}, **kwargs):
+def send_email(
+    subject,
+    body_text,
+    body_html,
+    recipients,
+    extra_context=None,
+    html_email_template=None,
```

Comment:
Why not:
```suggestion
    html_email_template=app_settings.OPENWISP_EMAIL_TEMPLATE,
```

Is this on purpose or not?